Doc/library/sqlite3.rst

```diff
@@ -1975,6 +1976,16 @@ both styles:
 
    [('C', 1972)]
 
+.. note::
+
+   :pep:`249` numeric placeholders are not supported.
```

Comment:
this works.

Reply:
Yeah, I guess keeping this note and removing the sentences below should be fine. Thanks for reviewing!